docs/content/en/docs/operator-manual/troubleshoot_learning.md

```diff
@@ -40,7 +40,7 @@ You can use the session ID to track whether learning occured.
 
 ```bash
 CONTEXTID=01J7S3QZMS5F742JFPWZDCTVRG
-curl -s -X POST https://localhost:8877/foyle.logs.SessionsService/GetSession -H "Content-Type: application/json" -d "{\"contextId\": \"${CONTEXTID}\"}" | jq.
+curl -s -X POST http://localhost:8877/api/GetSession -H "Content-Type: application/json" -d "{\"contextId\": \"${CONTEXTID}\"}" | jq .
```

Comment:
It should be 
http://localhost:8877/api/foyle.logs.SessionsService/GetSession

api - is the base prefix for the connect services
foyle.logs.SessionsService is the name of the service
GetSession is the method name